docs/docs/Introduction/intro.md

```diff
@@ -0,0 +1,52 @@
+---
+description: Introduction to FlutterFlow.
+---
+
+# Introduction
```

Comment:
This will make the sidebar titles as 
```
Introduction
    - Introduction 
```
So shall we update the name to something else to avoid repeatability. 
```
Introduction
    - Introduction to FlutterFlow (or something else?)
```
    
To do this, we can add front matter above like you did for description 

``` 
---
title: Introduction to FlutterFlow
description: Introduction to FlutterFlow.
---
```